velox/experimental/cudf/tests/HashJoinTest.cpp

```diff
@@ -1123,35 +1134,37 @@ TEST_P(MultiThreadedHashJoinTest, nullAwareAntiJoinWithFilter) {
             });
       });
 
-  HashJoinBuilder(*pool_, duckDbQueryRunner_, driverExecutor_.get())
-      .numDrivers(numDrivers_)
-      .probeKeys({"t0"})
-      .probeVectors(std::move(probeVectors))
-      .buildKeys({"u0"})
-      .buildVectors(std::move(buildVectors))
-      .joinType(core::JoinType::kAnti)
-      .nullAware(true)
-      .joinFilter("t1 != u1")
-      .joinOutputLayout({"t0", "t1"})
-      .referenceQuery(
-          "SELECT t.* FROM t WHERE NOT EXISTS (SELECT * FROM u WHERE t0 = u0 AND t1 <> u1)")
-      .checkSpillStats(false)
-      .verifier([&](const std::shared_ptr<Task>& task, bool /*unused*/) {
-        // Verify spilling is not triggered in case of null-aware anti-join
-        // with filter.
-        const auto statsPair = taskSpilledStats(*task);
-        ASSERT_EQ(statsPair.first.spilledRows, 0);
-        ASSERT_EQ(statsPair.first.spilledBytes, 0);
-        ASSERT_EQ(statsPair.first.spilledPartitions, 0);
-        ASSERT_EQ(statsPair.first.spilledFiles, 0);
-        ASSERT_EQ(statsPair.second.spilledRows, 0);
-        ASSERT_EQ(statsPair.second.spilledBytes, 0);
-        ASSERT_EQ(statsPair.second.spilledPartitions, 0);
-        ASSERT_EQ(statsPair.second.spilledFiles, 0);
-        verifyTaskSpilledRuntimeStats(*task, false);
-        ASSERT_EQ(maxHashBuildSpillLevel(*task), -1);
-      })
-      .run();
+  ASSERT_THROW(
```

Comment:
Usig VELOX_ASSERT_THROW would allow to specify an error message, which (1) helps ensure that the failure is an expected one, not a new / unexpected and serve as documentation for what's now working and how.